frontend/packages/dev-console/src/components/helm/helm-utils.ts

```diff
@@ -123,25 +124,29 @@ export const getChartEntriesByName = (
 export const concatVersions = (
   chartVersion: string,
   appVersion: string,
+  t: TFunction,
   chartRepoName?: string,
 ): string => {
   let title = chartVersion.split('--')[0];
   if (appVersion) {
-    title += ` / App Version ${appVersion}`;
+    title += t('devconsole~ / App Version {{appVersion}}', { appVersion });
   }
   if (chartRepoName) {
-    title += ` (Provided by ${toTitleCase(chartRepoName)})`;
+    title += t('devconsole~ (Provided by {{toTitleCase}})', {
+      toTitleCase: toTitleCase(chartRepoName),
```

Comment:
Please give this localization variable a cleaner name, `chartRepoName` or `repoName`.

Needs to be changed on some tests as well